projects.qmd

```diff
@@ -1,5 +1,502 @@
+
+All formats specifed in each file will be rendered.
+
+You can also specify exactly what you do and do not want rendered with the `render` key in `_quarto.yml`. See [Render Targets](https://quarto.org/docs/projects/quarto-projects.html#render-targets) in the Quarto documentation for more information.
+
+::: 
+
+To preview a project when using the `default` project type, you must specify a file:
+
+```{.bash filename="Terminal"}
+quarto preview index.qmd
+```
+
+Quarto will render just enough of the project to display the requested document and open a preview. 
+As you interact with the preview, e.g. follow a link in one document to another, Quarto will render any additional files as needed.
+
+## Project Configuration --- `_quarto.yml`
+
+The file `_quarto.yml` provides project configuration. Typically it will include project-level options, project options specific to the project type, and document-level options common to documents in the project.
+
+Here's a typical example:
+
+```{.yaml filename="_quarto.yml"}
+project:                         #<1>
+  type: website                  #<1>
+  title: "Quarterly Reports"     #<1>
+
+website:                         #<2>
+  navbar:                        #<2>
+    search: true                 #<2>
+
+execute:                         #<3>
+  echo: false                    #<3>
+
+format:                          #<3>
+  html:                          #<3>
+    toc: true                    #<3>
+```
+1. Project level options are set under `project`. You'll learn more about these in @sec-project-options.
+2. Project type specific options are set under the option that matches the type, e.g. `website`. You learn about these in the later chapters on the different project types.
+3. Document-level defaults that apply to all documents in the project. You'll learn more about these below in @sec-common-metadata.
+
+If you are curious what a more complicated `_quarto.yml` might look like, you could look at the [`_quarto.yml` file on GitHub](https://github.com/mine-cetinkaya-rundel/quarto-tdg/blob/main/_quarto.yml) for the project that builds this book.
+
+### Project Level Options {#sec-project-options}
+
+Project level options are set under the `project` key. You've already seen projects might have a `type` and `title`. 
+
+``` {.yaml filename="_quarto.yml"}
+project:
+  type: default
+  title: Data Exploration 
+```
+
+Another commonly set option is `output-dir` which specifes a location for rendered output. 
+
+Without setting `output-dir` outputs are rendered alongside their source documents. 
+For example, as you saw in the example in @sec-overview, the output `01-import.html` is rendered in the same location as `01-import.qmd`:
+
+```{r}
+dir_tree(dir, recurse = 1)
+```
+
+To share results without sharing the source documents, it may be easier to have your outputs in their own directory. 
```

Comment:
I'm not sure I understand this sentence. Maybe "it may be easier to have your outputs be written to a separate directory"?